Automattic/vip-cli
# Swap isomorphic-fetch for node-fetch

node-fetch is smaller and cleaner to work with. We also don't need most of the benefits of isomorphic-fetch.

Fixes #335

## Steps to Test

1. Check out PR.
1. Run `npm run build`
1. Run various commands to make sure they all work: 

- `DEBUG=* node ./dist/bin/vip-app 309`
- `DEBUG=* node ./dist/bin/vip-wp @1513 -- option get home`
- `DEBUG=* node ./dist/bin/vip-app-list`
